src/mysql/migrate.test.ts

```diff
@@ -44,8 +43,8 @@ describe('Migrate', () => {
 
   it('should migrate schema over two migration runs', async () => {
     const initialMigrate = await doInitialMigrate()
-    const [migrationResultBefore, timingBefore] = await time(initialMigrate.migrate('2020-04-02T165700'))
-    console.log(timingBefore / 1000)
+    const [migrationResultBefore] = await time(initialMigrate.migrate('2020-04-02T165700'))
```

Comment:
What about leaving the `timingBefore` and `timingAfter` variables in place (and adding a lint ignores)? That would make it easier to uncomment the commented timing logs. Otherwise one would have to re-add the variables in the right places when uncommenting the `console.log`'s.

Reply:
Makes sense. I've re-added them.